source/documentation/appendix-docs/botor.md

```diff
@@ -505,6 +508,8 @@ write_file_to_s3("my_local_data.csv", "s3://alpha-mybucket/my_data.csv")
 
 #### `list_files_in_buckets`
 
+The libraries `dplyr` and `purrr` need to be installed.
```

Comment:
Should we move dependencies to the top of the s3tools conversion section? I guess we could just have a list of `renv::install()` calls for people to run

Reply:
I was thinking that people might only want to use a subset of the functions so didn't want them to install anything unnecessarily.

Reply:
That sounds like a reasonable compromise. Happy to approve